### ISSUE TITLE
Stops loading in 1.4.2 and 1.4.3 and 1.4.1

### ISSUE THREAD

**Issue by Delliardo** [shadowmage45/SSTULabs#713]
I downloaded the Dev Repository as suggested in the forum post, and installed the required items from the gamedata folder.  https://i.imgur.com/Na0a2Zr.png

KSP stops loading part way through.  KSP.log from 1.4.2 attempt https://www.dropbox.com/s/u7x88kbq544y3ar/KSP.log?dl=0

**Comment by Jimbodiah** [comment]
use MM 3.0.6  or 3.0.7  as 3.0.1 is an outdated version even for KSP 1.3

https://ksp.sarbian.com/jenkins/job/ModuleManager/


Maybe Mage can update this in the Dev repository. I always use the latest version regardless of what mods include, and had sstu running yesterday (have nto tried any of todays builds).

**Comment by Delliardo** [comment]
Just tried MM 3.0.7 in my 1.4.2 install, and same problem.

**Comment by shadowmage45** [comment]
Something is causing textures-unlimited to fail to load in your install -- 
````
[EXC 18:23:45.103] TypeLoadException: Could not load type 'KSPShaderTools.TexturesUnlimitedLoader' from assembly 'SSTUTools'.
````

Specifically, it is crashing out when attempting to load one of the HAB parts:
````

[LOG 18:23:56.593] Error while creating AssetBundle request: System.UriFormatException: Invalid URI: Invalid port number
  at System.Uri.Parse (UriKind kind, System.String uriString) [0x00000] in <filename unknown>:0 
  at System.Uri.ParseUri (UriKind kind) [0x00000] in <filename unknown>:0 
  at System.Uri..ctor (System.String uriString, Boolean dontEscape) [0x00000] in <filename unknown>:0 
  at System.Uri..ctor (System.String uriString) [0x00000] in <filename unknown>:0 
  at UnityEngineInternal.WebRequestUtils.MakeInitialUrl (System.String targetUrl, System.String localUrl) [0x00000] in <filename unknown>:0 
[EXC 18:23:56.594] NullReferenceException: Object reference not set to an instance of an object
	SSTUTools.SMFBundleDefinitionReader+<Load>d__0.MoveNext ()
	UnityEngine.SetupCoroutine.InvokeMoveNext (IEnumerator enumerator, IntPtr returnValueAddress)
````

**Comment by shadowmage45** [comment]
Looks like the dev branch had an out-of-date version of TexturesUnlimited stored in it.  I've updated the dev branch to have the most recent TexturesUnlimited, but it might still have other out-of-date dependencies.

(so, you should re-download the .zip, and/or update your TU install manually)

Edit:  I'll have to do a check on the rest of the deps, as I don't think I've updated any of those in the SSTU dev repo either... (module-manager, KSPWheel, CRP)

**Comment by Delliardo** [comment]
Looks like the updated Dev Rep is fixed, got it working in 1.4.2!
